packages/block-library/src/button/block.json

```diff
@@ -9,6 +9,14 @@
 	"keywords": [ "link" ],
 	"textdomain": "default",
 	"attributes": {
+		"tagName": {
+			"type": "string",
+			"default": "a"
+		},
```

Comment:
This should probably be an `enum` as we only want to allow `a` and `button` elements.